src/features/dataset/CompletionCheckmark.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+
+interface CompletionChenckmarkProps {
+  type: 'green' | 'gray' | 'orange' | 'outline'
+  className?: string
+}
+
+const CompletionCheckmark: React.FC<CompletionChenckmarkProps> = ({ type, className }) => {
```

Comment:
2 things: 
1) Is this used anywhere?
2) we seem to have two components called `CompletionCheckmark`, one defined in `src/features/dataset` & one defined in `src/features/datasetEditor`

Reply:
I believe these are for the "tic tacs" for showing which components have changed.  We aren't yet ready to use these but we can clean this up later